recipes/automake/all/test_package/conanfile.py

```diff
@@ -49,7 +50,9 @@ def _build_scripts(self):
 
     def _build_autotools(self):
         """Test autoreconf + configure + make"""
-        self.run("{} --verbose --install".format(os.environ["AUTORECONF"]), win_bash=tools.os_info.is_windows)
+        with tools.environment_append({"AUTOMAKE_CONAN_INCLUDES": [tools.unix_path(self.source_folder)]}):
+            print(os.environ["AUTOMAKE_CONAN_INCLUDES"])
```

Comment:
Use `self.output.info` if this is something you want to print in the build process

Reply:
This is a remainder of a debug session. It can be removed.